python/dgl/data/__init__.py

```diff
@@ -11,14 +11,20 @@
 from .ppi import PPIDataset
 from .tu import TUDataset
 from .gindt import GINDataset
-from .chem import Tox21
+from .chem import Tox21, alchemy
```

Comment:
Let's be consistent here. First add `from .alchemy import TencentAlchemyDataset` in `data/chem/__init__.py` and then `from .chem import Tox21, TencentAlchemyDataset`.